adafruit_ble/__init__.py

```diff
@@ -156,9 +156,12 @@ class BLERadio:
     It uses this library's `Advertisement` classes and the `BLEConnection` class."""
 
     def __init__(self, adapter=None):
-        if not adapter:
-            adapter = _bleio.adapter
-        self._adapter = adapter
+        """If no adapter is supplied, use the built-in _bleio.adapter, if available.
+        If not, see if there's an onboard HCI adapter available that could be used.
```

Comment:
> If not, see if there's an onboard HCI adapter available that could be used.

This code doesn't actually do this right? I think it'd be easier to explain that this will raise an error.

Reply:
Right, it used to do that, and I forgot to change the doc. :)